Update `insar_tops` and `insar_tops_burst` to grab orbit data from CDSE (#160)
ASFHyP3/hyp3-isce2: README.md

> ### Earthdata Login

Reviewer: ```suggestion
### Earthdata Login and ESA Credentials
```

For all workflows the user will need to provide their Earthdata login credentials, and an ESA Copernicus Data Space Ecosystem (CDSE) to download input data. 
If you do not already have an Earthdata account, you can sign up [here](https://urs.earthdata.nasa.gov/home). 

> If you do not already have an CDSE account, you can sign up [here](https://dataspace.copernicus.eu). 

Reviewer: ```suggestion
If you do not already have a CDSE account, you can sign up [here](https://dataspace.copernicus.eu). 
```

> Your credentials can either be passed to the workflows via environment variables 

Reviewer: ```suggestion
Your credentials can be passed to the workflows via command-line options (`--esa-username` and  `--esa-password`), environment variables 
```
